exe/kubernetes-deploy

```diff
@@ -43,16 +45,22 @@ revision = ENV.fetch('REVISION') do
   exit 1
 end
 
-KubernetesDeploy::Runner.with_friendly_errors do
-  runner = KubernetesDeploy::Runner.new(
-    namespace: ARGV[0],
-    context: ARGV[1],
-    current_sha: revision,
-    template_dir: template_dir,
-    wait_for_completion: !skip_wait,
-    allow_protected_ns: allow_protected_ns,
-    prune: prune,
-    bindings: bindings
-  )
-  runner.run
-end
+namespace = ARGV[0]
+context = ARGV[1]
+logger = KubernetesDeploy::FormattedLogger.build(namespace, context, verbose_prefix: verbose_log_prefix)
+
+runner = KubernetesDeploy::Runner.new(
+  namespace: namespace,
+  context: context,
+  current_sha: revision,
+  template_dir: template_dir,
+  bindings: bindings,
+  logger: logger
+)
+
+success = runner.run(
+  verify_result: !skip_wait,
```

Comment:
The actual method in `Runner` that does the waiting is called `wait_for_completion`, and now that this is an arg to `Runner.run`, it ends up being a local var. So you have both a local var and a method called `wait_for_completion` being used on adjacent lines. I don't feel as strongly as you about this naming, so I'll use whatever you want. But I need two names: one for the flag and one for the method.